src/debug/elf.cr

```diff
@@ -168,7 +168,7 @@ module Debug
       ei_version = @io.read_byte.not_nil!
       raise Error.new("Unsupported version number") unless ei_version == 1
 
-      ei_osabi = OSABI.from_value(@io.read_byte)
+      ei_osabi = OSABI.from_value(@io.read_byte.not_nil!)
```

Comment:
Should we `raise IO::EOFError.new` here instead of `not_nil!` in the unfortunate case where `read_byte` is nil?

Reply:
@ysbaddaden ^

Reply:
Everywhere around here uses `not_nil!`, any change would need to change all of them in another PR.

Reply:
ah indeed, no question then